### PR TITLE
PRO-5533: stop update to case data after payment attempt

### PR DESCRIPTION
**Pull request by Wivenhoe-HMCTS**
PRO-5533: stop update to case data after payment attempt
```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: test/component/payment/testPaymentBreakdown.js [hunk 1]
```diff
@@ -45,7 +45,10 @@ describe('paymentBreakdown', () => {
         testHelpBlockContent.runTest('PaymentBreakdown');
 
         it('test right content loaded on the page', (done) => {
-            const sessionData = {applicant: 'value'};
+            const sessionData = {applicant: {
+                firstName: 'value'
+            }
+            };
```
Comment (LucaDelBuonoHMCTS) [inline test/component/payment/testPaymentBreakdown.js]: Ugly indentation :)

File: test/component/payment/testPaymentStatus.js [hunk 2]
```diff
@@ -37,7 +37,10 @@ describe('paymentStatus', () => {
         });
 
         it('test right content loaded on the page', (done) => {
-            const sessionData = {applicant: 'value'};
+            const sessionData = {applicant: {
+                firstName: 'value'
+            }
+            };
```
Comment (LucaDelBuonoHMCTS) [inline test/component/payment/testPaymentStatus.js]: Ugly indentation :)

File: test/component/summary/testDeceasedSection.js [hunk 3]
```diff
@@ -24,7 +24,10 @@ describe('summary-deceased-section', () => {
 
     describe('Verify Content, Errors and Redirection', () => {
         it('test correct content loaded on deceased section of the summary page , when no data is entered', (done) => {
-            const sessionData = {applicant: 'value'};
+            const sessionData = {applicant: {
+                firstName: 'value'
+            }
+            };
```
Comment (LucaDelBuonoHMCTS) [inline test/component/summary/testDeceasedSection.js]: Ugly indentation :)

File: test/component/summary/testSummary.js [hunk 4]
```diff
@@ -33,7 +33,10 @@ describe('summary', () => {
         testHelpBlockContent.runTest('Summary');
 
         it('test content loaded on the page', (done) => {
-            const sessionData = {applicant: 'value'};
+            const sessionData = {applicant: {
+                firstName: 'value'
+            }
+            };
```
Comment (LucaDelBuonoHMCTS) [inline test/component/summary/testSummary.js]: Ugly indentation :)

File: app/routes.js [hunk 5]
```diff
@@ -57,13 +58,23 @@ router.use((req, res, next) => {
...
+router.get('/*', (req, res, next) => {
+    const formdata = req.session.form;
+    if (!includes(config.whiteListedPagesForPaymentBreakdown, req.originalUrl) &&
+        get(formdata, 'ccdCase.id', '') !== '') {
+        res.redirect(`${config.app.basePath}/payment-breakdown`);
+    } else {
+        next();
+    }
+});
+
```
Comment (LucaDelBuonoHMCTS) [inline app/routes.js]: See if you can have these 3 blocks of code inside the same
```
router.get('/*', (req, res, next) => {
   ...
});
```

File: test/component/payment/testPaymentBreakdown.js (hunk 1 above)
Comment (Wivenhoe-HMCTS) [reply test/component/payment/testPaymentBreakdown.js]: done

File: test/component/payment/testPaymentStatus.js (hunk 2 above)
Comment (Wivenhoe-HMCTS) [reply test/component/payment/testPaymentStatus.js]: done

File: test/component/summary/testDeceasedSection.js (hunk 3 above)
Comment (Wivenhoe-HMCTS) [reply test/component/summary/testDeceasedSection.js]: done

File: test/component/summary/testSummary.js (hunk 4 above)
Comment (Wivenhoe-HMCTS) [reply test/component/summary/testSummary.js]: done

**Pull request merged**